docs/typescript/clients.md

```diff
@@ -246,3 +259,97 @@ export async function CancelExternalWorkflow(wfId: string): void {
 ```
 
 Again, see the [Workflow APIs documentation](/docs/typescript/workflows#external-workflows) for full details.
+
+## Advanced: Making raw gRPC calls
+
+Under the hood of a `WorkflowClient`, the `Connection` is actually powered by a `WorkflowService` driver that makes the raw gRPC calls to Temporal Server.
+This Service is capable of making a wider range of introspection calls (as per [the API reference](https://typescript.temporal.io/api/classes/proto.temporal.api.workflowservice.v1.WorkflowService-1#methods)):
+
+```ts
```

Comment:
I'd really rather stop putting snippets directly into the docs, we have snipsync, let's use it so we know the snippets work.

Reply:
yeah ok ok on it